src/sentry/backup/imports.py

```diff
@@ -65,6 +65,7 @@
     "sentry.team": {"org_role"},
     # TODO(mark): Safe to remove after july 2024 after self-hosted 24.6.0 is released
     "sentry.rule": {"owner"},
+    "sentry.alertrule": {"owner"},
```

Comment:
nit: would be nice to note that the above TODO applies here too, or add a separate copy.